### DISCUSSION TITLE
Mobile WebAuthn connector page garbled

### DISCUSSION THREAD

**Discussion by cherinyy** [dani-garcia/vaultwarden#2908] [Q&A]
Hello,

I know it's a client issue, but I'm not quite sure how it happens.

https://github.com/bitwarden/clients/issues/3846

When I login from the Bitwarden Android client, the Chinese characters in the Mobile WebAuthn connector page under the Chrome Custom Tab are garbled.

![Screenshot_2022-10-20-11-49-17-03_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/9050713/196857404-acd627aa-085a-455d-9335-ab9f8b239c39.jpg)

Then when I logged in successfully, the "avatar" generated by my nickname was also not displayed accurately. Show as Specials.

This issue does not appear in the Windows client, Chrome extensions, or Firefox addon.

**Comment by BlackDex** [comment]
Looks like an UTF8/UTF16 issue here.
Maybe your server returns something else then UTF8? Or the browser expects something else?
That could garble the output.
